### PR TITLE
#patch Release v0.7.11

### PR DESCRIPTION
**Pull request by sheridavandenbent**
# Results of merging this
Additions in this version:
- The trade log has been updated to include general info like timeframe, max open trades, and strategy name
- Various small improvements and code efficiencies

### CODE REVIEW

**Review by marijn111 — APPROVED** [review]
Nice!

File: README.md [hunk 1]
```diff
@@ -29,109 +31,45 @@ may have.
...
 
-2. Install TA-lib library by running this command:
-```
-brew install ta-lib
-```
-OR if you you skipped Step 1. you can run this command (requires [Xcode12](https://developer.apple.com/download/)):
-```
-pip install ta-lib
-```
+# Running the engine
```
Comment (DemianVoorhagen6) [inline README.md:56]:
```suggestion
# Running the Engine
```

Comment (DemianVoorhagen6) [reply README.md:56]: Maybe cmd+f for all 'engine' > 'Engine'?

**Review by DemianVoorhagen6 — COMMENTED** [review]
Looks good overall, I did not check the exact trade logic changes but from gut feeling it looked right :)

File: README.md (hunk 1 above)
Comment (sheridavandenbent) [reply README.md]: Done!

**Pull request merged**